x/gateway/types/message_unstake_gateway.go

```diff
@@ -39,7 +39,7 @@ func (msg *MsgUnstakeGateway) GetSignBytes() []byte {
 func (msg *MsgUnstakeGateway) ValidateBasic() error {
 	_, err := sdk.AccAddressFromBech32(msg.Address)
 	if err != nil {
-		return sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress, "invalid address address (%s)", err)
+		return sdkerrors.Wrapf(ErrGatewayInvalidAddress, "invalid address address (%s)", err)
```

Comment:
I would avoid these changes as they are addressed in #75 as these are from the scaffolding alone addressing them here seems out of scope of this PR

Reply:
I already have it in place and I think doing a `git merge` + conflict resolution would be easier than checking out all the places I need to revert.